src/main/scala/clump/ClumpSource.scala

```diff
@@ -16,14 +15,30 @@ class ClumpSource[T, U](val fetch: Set[T] => Future[Map[T, U]], val maxBatchSize
 
   def get(input: T): Clump[U] =
     new ClumpFetch(input, ClumpContext().fetcherFor(this))
+
+  def maxBatchSize(int: Int) =
```

Comment:
It is possible to follow the same approach if we introduce other configurations for source instances.